.config/.tailwind/presets.js

```diff
@@ -49,17 +49,34 @@ const spacing = {
   xl: '48px',
 };
 
+// const screens = {
+//   xs: { min: 'var(--screen-xs-min)', max: 'var(--screen-xs-max)' },
+//   sm: { min: 'var(--screen-sm-min)', max: 'var(--screen-sm-max)' }, 
+//   md: { min: 'var(--screen-md-min)', max: 'var(--screen-md-max)' },
+//   lg: { min: 'var(--screen-lg-min)', max: 'var(--screen-lg-max)' },
+//   xl: 'var(--screen-xl-min)',
+// };
+
+// const screens = {
+//   xs: { min: '0px', max: '639px' },
+//   sm: { min: '640px', max: '767px' }, 
+//   md: { min: '768px', max: '1023px' },
+//   lg: { min: '1024px', max: '1440px' },
+//   xl: '1920px',
+// };
+
 const screens = {
```

Comment:
this means 
'sm': '767px' = @media (min-width: 767px)

but our breakpoints is supposed to be:
![image](https://github.com/Messagepool/hn-gel/assets/101383236/0f21fe47-f6cd-4ff0-8760-e0f3f43e204b)